### ISSUE TITLE
Keyboard navigation in Flex control

### ISSUE THREAD

**Issue by rpominov** [webstudio-is/webstudio-designer#736]
I've created this issue after a discussion in https://github.com/webstudio-is/webstudio-designer/pull/723#discussion_r1060713298 , but I'm not completely sure I understand the task. Here's how I understand it:

- We want to enable keyboard navigation in this part of the flex controls: <br/><img width="88" alt="Screenshot 2023-01-05 at 11 50 34" src="https://user-images.githubusercontent.com/825702/210728755-e19b458e-9805-43c4-84b1-4e1c8cad805e.png">
- It should work similarly to Spacing / [Space](https://github.com/webstudio-is/webstudio-designer/issues/733): https://github.com/webstudio-is/webstudio-designer/issues/666
- Enter should have the same effect as click.

Open questions:

1. How focused state should look like? Currently the outline seem to be used to distinguish between set / unset / remote / etc <br/><img width="88" alt="Screenshot 2023-01-05 at 11 56 19" src="https://user-images.githubusercontent.com/825702/210729785-fc988227-d9ac-465b-9e90-15190903476a.png">
2. When the selected items are stretched like that, how should we highlight these selected items when user navigates with arrow keys? The hover effect is very hard to see in this case (in the screenshot the top middle item is hovered) <br/><img width="85" alt="Screenshot 2023-01-05 at 11 58 33" src="https://user-images.githubusercontent.com/825702/210730203-5139d777-f157-44ae-9ff6-a6b2367c9f4e.png">

**Comment by rpominov** [comment]
@kof @taylornowotny What do you think about the questions above?

**Comment by kof** [comment]
We would need to have some focus state as we have in spacing for internal elements @taylornowotny 
Also if this is not-trivial (more than a day of work), we should do this after public beta launch.

**Comment by taylornowotny** [comment]
Good news - this control does not need to be keyboard accessible!

Keyboard users can access all the same controls via the icon buttons to the right. The flex control is totally redundant with those buttons, so a keyboard user will never be blocked by lacking access to the flex control. It's just there as a nice shortcut for mouse users and as a visualization of flexbox (becuase it can be confusing to new users and sometimes flex settings aren't evident from the canvas).

**Comment by taylornowotny** [comment]
<img width="807" alt="Screenshot 2023-01-05 at 8 05 16 AM" src="https://user-images.githubusercontent.com/86495896/210825892-d80b542d-76fe-4911-9c05-6bbb0c38ff3f.png">

From the docs

**Comment by kof** [comment]
> Good news - this control does not need to be keyboard accessible!
True, all we need for now is to fix the outline for that grid control,menu buttons there are already accessible

**Comment by rpominov** [comment]
Why do we need to fix outline if we're not going to implement keyboard support and therefore don't need outline for the focus state?

**Comment by kof** [comment]
You should be able to tab through all controls, just try focusing the display block and then tab through flex controls, you will see something is off

**Comment by kof** [comment]
We have a terminology problem: I meant the entire thing: 
![image](https://user-images.githubusercontent.com/52824/210841585-008d439c-c25f-4d33-b30d-42e4e3c1a66e.png)
should be one tab and have an outline, then one would move on to the menu buttons with the next tab

**Comment by taylornowotny** [comment]
I understood you. What is the purpose of these flex controls being tab-able?

**Comment by taylornowotny** [comment]
@rpominov I talked with Oleg to clear this up.

This part of the UI is now called "flex control grid" to avoid confusing it with any other UI
<img width="508" alt="Screenshot 2023-01-06 at 9 52 31 AM" src="https://user-images.githubusercontent.com/86495896/211071161-2c3cdc94-7b79-4142-983d-e25ca4e66b87.png">
<img width="503" alt="Screenshot 2023-01-06 at 9 59 36 AM" src="https://user-images.githubusercontent.com/86495896/211071117-3735ae8f-e13e-4c98-96ec-411fbf646e49.png">

The flex control grid should NOT be keyboard accessible in any way. Not tab-able/keyboard focusable.

The rest of the Display section should be keyboard accessible as normal.

Let me know if there is anything confusing about this.

**Comment by kof** [comment]
So we would the Flex Control Grid basically tabIndex -1, you would jump from display to the first menu item directly
